src/SoundPlayer.cpp

```diff
@@ -151,4 +154,41 @@ ZunResult SoundPlayer::LoadWav(char *path)
     return ZUN_SUCCESS;
 }
 
+ZunResult SoundPlayer::InitSoundBuffers()
+{
+    i32 idx;
+    if (this->manager == NULL)
+    {
+        return ZUN_ERROR;
+    }
+    else if (this->dsoundHdl == NULL)
+    {
+        return ZUN_SUCCESS;
+    }
+    else
+    {
+        for (idx = 0; idx < 3; idx++)
+        {
+            this->soundBuffersToPlay[idx] = -1;
+        }
+        for (idx = 0; idx < ARRAY_SIZE_SIGNED(g_SFXList); idx++)
+        {
+            if (this->LoadSound(idx, g_SFXList[idx]) != ZUN_SUCCESS)
+            {
+                GameErrorContextLog(&g_GameErrorContext, TH_JP_ERR_SOUNDPLAYER_FAILED_TO_LOAD_SOUND_FILE,
```

Comment:
```suggestion
                GameErrorContextLog(&g_GameErrorContext, TH_ERR_SOUNDPLAYER_FAILED_TO_LOAD_SOUND_FILE_FILE,
```

Reply:
This isn't fixed. Did you forget to push?

Reply:
probably